### PR TITLE
chore: update api docs link

### PR DESCRIPTION
**Pull request by deekshas8**
This PR fixes the links for docs portal and api docs in README files.

### CODE REVIEW

File: packages/connectivity/README.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ If you would like to contribute to the SAP Cloud SDK, please make yourself famil
 
 <br>
 
-- [SAP Cloud SDK Documentation portal](https://sap.github.io/cloud-sdk/api/latest)
+- [SAP Cloud SDK Documentation portal](https://sap.github.io/cloud-sdk)
 - [SAP Cloud SDK Documentation portal - Getting started guide](https://sap.github.io/cloud-sdk/docs/js/getting-started)
-- [SAP Cloud SDK Documentation portal - API documentation](https://sap.github.io/cloud-sdk/docs/js/api-reference-js-ts)
+- [SAP Cloud SDK Documentation portal - API documentation](https://sap.github.io/cloud-sdk/api/latest)
```
Comment (FrankEssenberger) [inline packages/connectivity/README.md:46]: [q] What if the java sdk wanted to host their JavaDoc somewhere? The URL does not contain the `js` part? @marikaner should we change this in the future?

Comment (marikaner) [reply packages/connectivity/README.md:46]: If we change this in the future, we should change it here as well, but for now we can leave it. I guess if we change this we will also add another redirect.

**Review by marikaner — APPROVED** [review]
LGTM

**Review by FrankEssenberger — APPROVED** [review]
LGTM - just a general question

**Pull request merged**